### PR TITLE
Accessiblity Swarm info in Contributing.md

### LINKED ISSUE #103: Add Accessibility Swarm process to Psammead Contributing guide
**Is your feature request related to a problem? Please describe.**
We currently do not refer to our Accessibility swarm process within Psammead.

We should add a `Accessibility_Swarm.md`, that can be referenced in the `Contributing.md` file. 

**Describe the solution you'd like**
Add a `Accessibility_Swarm.md`, which explains what happens during an accessibility swarm focusing on a new component. It should include 
- Disciplines involved and the process
- What the checklist of tasks is. Turning the tables here into markdown: ( )
- How we prioritise testing on certain devices and Assistive Technologies by referring to the categorisation of Assistive Tech here: https://github.com/BBC-News/psammead/issues/99
- What the outcomes of the swarm are.

This write-up should be in the Psammead repository and not just linking to external resources. 

**Describe alternatives you've considered**
If this information existed on https://bbc-news.github.io/accessibility-news-and-you/, we could just refer to it. However, it currently does not exist on this site.  

@greenc05 has a slideshow version introducing an Accessibility Swarm. It would be good to contact her before picking up this issue. 

**Testing notes**
[Tester to complete]

Dev insight: This is documentation. It should be easily understood by Devs, Testers, Product Owners, UX Designers & Business Analysts. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #103

Adding accessiblity swarm information to contributing guide

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,7 +111,8 @@ In addition, the `README.md` should also point to Psammead's [CODE_OF_CONDUCT](h
 ### After merge
 
 After merging a pull request into `latest`, please follow these steps to ensure that the changes are published to npm and to the Psammead Storybook https://bbc-news.github.io/psammead.
+
 - Publish Packages  
-  To make your changes available on the npm registry please publish any package changes. The instructions to do this can be found in the the [repository README](https://github.com/BBC-News/psammead#rocket-publishing-packages) 
+  To make your changes available on the npm registry please publish any package changes. The instructions to do this can be found in the the [repository README](https://github.com/BBC-News/psammead#rocket-publishing-packages)
```
[the remark this answers is not in the document]
Comment (radiocontrolled) [reply CONTRIBUTING.md]: I also appreciate that publishing is not what your PR is about.

**Review by radiocontrolled — APPROVED** [review]
Ignore my comment -- I late afternoon misread and thought you were making a documentation change re: publishing process. Although it points to the main readme, which points out packages are published on merge to `latest`, the language is a bit unclear from the CONTRIBUTING.md section on publishing. Not an issue for this issue though.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bcmn) [inline CONTRIBUTING.md:116]: I don't think this was an intended change of your PR, but is either of these sections ("publish packages", "publish storybook") still necessary? CI handles this for us & we document that process elsewhere.

Comment (ChrisBAshton) [reply CONTRIBUTING.md:116]: Agreed, we should remove - let's raise another PR for that

**Review by ChrisBAshton — APPROVED** [review]

**Comment by ChrisBAshton** [comment]
Merging documentation.

**Pull request merged**